developer-docs-site/docs/concepts/coin-and-token/aptos-token.md

```diff
@@ -270,24 +217,23 @@ We provide `burn` and `burn_by_creator` functions for token owners and token cre
 Burn is allowed only when the `BURNABLE_BY_OWNER` property is set to `true` in `default_properties`. Burn by creator is allowed when `BURNABLE_BY_CREATOR` is `true` in `default_properties`.
 Once all the tokens belonging to a `TokenData` are burned, the `TokenData` will be removed from the creator’s account. Similarly, if all `TokenData` belonging to a collection are removed, the `CollectionData` will be removed from the creator’s account.
 
-## Token transfer
+### Token transfer
 
-To protect a user from receiving undesired NFTs, an Aptos user must be first offered an NFT, followed by the user claiming the offered NFTs. Then only these NFTs will be deposited in the user's token store. This is the default token transfer behavior. For example:
+We provide three different modes for transferring tokens between the sender and receiver.
 
-1. If Alice wants to send Bob an NFT, she must first offer Bob this NFT. This NFT is still stored under Alice’s account. 
-1. Only when Bob claims the NFT, this NFT will be removed from Alice’s account and stored in Bob’s token store. 
+Two-step transfer: to protect users from receiving undesired NFTs, they must be first offered NFTs, and then accept the offered NFTs. Then only the offered NFTs will be deposited in the users' token stores. This is the default token transfer behavior. For example:
+1. If Alice wants to send Bob an NFT, she must first offer Bob this NFT. This NFT is still stored under Alice’s account.
```

Comment:
Numbering looks wrong here

Reply:
This is automated:
https://github.com/aptos-labs/aptos-core/blob/46aec6b97c16a632a17461468cf3b40b4274d139/developer-docs-site/docs/concepts/coin-and-token/aptos-token.md